### ISSUE TITLE
No comments

### ISSUE THREAD

**Issue by deronsizemore** [InstaPy/InstaPy#1970]
## Expected Behavior

Bot will comment

## Current Behavior

I've set percentage to 100 for comments yet I watched it work for 15 minutes or so without commenting on a single post.

## InstaPy configuration

    # settings
    session.set_do_follow(True, percentage=80, times=1)
    session.set_lower_follower_count(limit=50)
    session.set_do_comment(enabled=True, percentage=100)
    session.set_comments([u'Nice shot! @{} :heart:', u'Really nice! @{} :thumbsup:', u'Loving your work! @{} :smile:', u'This is :fire: @{}', u'Great work! @{} :heart:', u'Love it! @{} :heart: :thumbsup:', u'Very nice! @{}'])

    # actions
    session.like_by_tags(like_tags, amount=100, interact=True)

Do I have my settings incorrect above? Commenting used to work when I had it coupled with set_user_interact, but that was problems with duplicate comments.

**Comment by uluQulu** [comment]
hi @deronsizemore,
`session.set_lower_follower_count` and `session.set_upper_follower_count` features are deprecated
why don't you pull latest commits and use `set_relationship_bounds` feature which has more functionalities? (_see `README` for usage and examples_)
_although, the words above does not affect your `commenting` misbehavior_
does other features work with commenting?

**Comment by deronsizemore** [comment]
Thanks. I got the latest commit working and it still doesn't appear to be commenting. There has to be something I'm missing here.

Here's my updated settings and actions:

    # settings
    session.set_relationship_bounds (
                    enabled=True,
                    potency_ratio=None,
                    delimit_by_numbers=True,
                    max_followers=None,
                    max_following=None,
                    min_followers=100,
                    min_following=None
                    )
    session.set_do_comment(True, percentage=100)
    session.set_comments([u'Nice shot! @{} :heart:', u'Really nice! @{} :thumbsup:', u'Loving your work! @{} :smile:', u'This is :fire: @{}', u'Great work! @{} :heart:', u'Love it! @{} :heart: :thumbsup:', u'Very nice! @{}'])

    # actions
    session.like_by_tags(like_tags, amount=100, interact=True)

Am I mistaken in thinking that this code should comment on 100% of the images that are liked? I sat at my machine and watched the automation work in the browser and it liked a few images but never commented on any.

**Comment by uluQulu** [comment]
Welcome, @deronsizemore,
That's right. It should comment on every liked image with the percentage of `100`.
Try this:
use `headless_browser=False` and watch the comenting activity on the web browser and see if it says `Commenting failed. Retry` error in bottom of screen

**Comment by deronsizemore** [comment]
Sorry for the delay. Newborn at home makes it difficult for computer fun time! 

I actually already had headless_browser=False in the settings. I didn't see anything in the browser window but I did just now notice in the terminal window: `--> Warning: Comment Button Not Found: May cause issues with browser windows of smaller widths`

Not sure what that's about. I stopped and started it again because it seems to just come to a hault when it experiences that warning. I expanded the browser window with no luck.

**Comment by deronsizemore** [comment]
I did see this: https://github.com/timgrossmann/InstaPy/issues/1117

Seems like maybe I need to update Chrome Driver so I'll try that as soon as I can

**Comment by deronsizemore** [comment]
No luck here. Just downloaded the latest Chrome Driver and ran Instapy. Just received: Warning: Comment Button Not Found: May cause issues with browser windows of smaller widths.

I tried maximizing my window and still nothing.

I also just received:

Warning: Comment Action Likely Failed: Comment Element not found

I went to that profile manually and there's no comments. 

I'm not sure where to go from here?

**Comment by uluQulu** [comment]
@deronsizemore 
Have you updated your selenium and Chrome browser, too?

**Comment by deronsizemore** [comment]
@uluQulu I have not updated Selenium but I have updated chrome browser and chrome driver. I'm confused on Selenium. I tried doing a pip install -U selenium but it tells me it's not installed. So I'm sure if I need to install it or what? I've been using Instapy for a year or so now without any issues. All I ever remember doing was downloading the .zip of the Instapy files, editing quickstart.py with  my settings an then running the quickstart.py file. It's worked to this point.

**Comment by uluQulu** [comment]
@deronsizemore 
If you can successfully start InstaPy instance then you have `selenium` installed
Maybe pip isn't configured work well on your system, also a screenshot of the console output will be helpful to troubleshoot easily.
> Installing or updating will do the same thing- grab the latest selenium release. But if you have already installed it, use update.

**Comment by deronsizemore** [comment]
@uluQulu Thanks. I'm not sure. PIP is working fine now (after having to update it). there was some sort of change with Python it seems and anytime I tried installing a package I was getting: 

> Could not fetch URL https://pypi.python.org/simple/selenium/: There was a problem confirming the ssl certificate: <urlopen error [SSL: TLSV1_ALERT_PROTOCOL_VERSION] tlsv1 alert protocol version (_ssl.c:590)>

So I've got Python and PIP updated and it still shows no installs of Selenium. All I did from the start was followed this: https://github.com/timgrossmann/InstaPy#set-it-up-yourself-with-this-basic-setup. I downloaded the .zip, extracted it, edited quickstart.py with my settings and then ran it via terminal using `python quickstart.py`. Maybe that's the wrong way to do it and I've just been getting lucky for the last half year or so using Instapy?

Anyway, I've completely set everything up again. I created a virtual environment and have InstaPy working inside of the environment with all required packages installed and I'm still getting:

> --> Warning: Comment Button Not Found: May cause issues with browser windows of smaller widths

I suppose I'll go start a new ticket for this one since it seems my issue has migrated over to this problem rather than the script simply not commenting.

Actually, I see another issue open regarding this so I'll follow along there.
